src/component/common/Table/TableSearch/TableSearchField/TableSearchFieldSuggestions/TableSearchFieldSuggestions.tsx

```diff
@@ -0,0 +1,202 @@
+import { FilterList } from '@mui/icons-material';
+import { Box, Divider, Paper, styled } from '@mui/material';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import {
+    getColumnValues,
+    getFilterableColumns,
+    getFilterValues,
+    getSearchTextGenerator,
+    IGetSearchContextOutput,
+} from 'hooks/useSearch';
+import { useMemo, VFC } from 'react';
+
+const randomIndex = (arr: any[]) => Math.floor(Math.random() * arr.length);
+
+const StyledPaper = styled(Paper)(({ theme }) => ({
```

Comment:
I think according to our discussion earlier these should be at the bottom of the file? Or did I misunderstand?

Reply:
I thinks so. I have a preference for putting it on top, to make sure main exported thing is always last. If the file is getting too long, I would put styled in separate one separate file.